<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <script async src="https://www.googletagmanager.com/gtag/js?id=G-4VQ0HG4VC8"></script> <script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-4VQ0HG4VC8');
</script> <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WRM6X7TS');</script> <meta name="google-site-verification" content="Agn48mofIJL-sVWqk7bJ2Ajms2VU5e6qiPG4rYXaPY8"> <meta http-equiv="Permissions-Policy" content="interest-cohort=()"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> RIMA | Amir Taheri-Ghahfarokhi </title> <meta name="author" content="Amir Taheri-Ghahfarokhi"> <meta name="description" content="Rational Indel Meta-Analysis"> <meta name="keywords" content="Amir, Taheri, Taheri-Ghahfarokhi, Gene-Editing, CRISPR, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/favicon.ico?5c06f3bd4ec2d683fcae74dc648ed392"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://amirtaheri.info/projects/01_RIMA/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Amir Taheri-Ghahfarokhi </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">Bio </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">Blog </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">Publications </a> </li> <li class="nav-item active"> <a class="nav-link" href="/projects/">Projects <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">Repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">Resume </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">RIMA</h1> <p class="post-description">Rational Indel Meta-Analysis</p> </header> <article> <p><strong>Background - “Random Indels”</strong><br> Genome editing involves two sequential steps: <em>cutting</em> and <em>repairing</em>. The <em>cut</em> is typically introduced by programmable nucleases, such as Cas9, in a targeted manner. The <em>repair</em> process, however, primarily depends on the cell’s own DNA repair mechanisms. If repair perfectly restores the sequence at the cleavage site, the targeted site may be cleaved repeatedly by the nuclease. However, repairs can also introduce new mutations, such as insertions and deletions (commonly referred to as indels), at the cleavage site.</p> <p><strong>The Challenge - DNA Repair Profiling</strong><br> Indels were long assumed to be random repair events. However, recent studies have demonstrated their non-random nature. Indel profiles at different genomic loci are distinct, yet these profiles are reproducible at the same target site across different cell lines and experimental replicates. This raises an intriguing question: What drives the formation of reproducible mutations at specific target sites in different cells?</p> <p><strong>The Achievement - Rational Indel Meta-Analysis (RIMA)</strong><br> During my research on characterizing Cas9 nucleases in human cells, I developed a bioinformatics tool called <em>Rational Indel Meta-Analysis (RIMA)</em> <a class="citation" href="#taheri2018decoding">(Taheri-Ghahfarokhi et al., 2018)</a>. RIMA analyzes, categorizes, and visualizes mutation patterns at Cas9 cleavage sites, shedding light on how DNA repair pathways interact with Cas9 activity and the sequence context of target sites. This tool helps explain the mechanisms underlying the non-random nature of Cas9-induced mutations.</p> <p>RIMA also profiles the contribution of different DNA repair pathways in the repair of DNA double-strand breaks. It has facilitated various applications in genome editing and cancer research. For example, I used RIMA to distinguish the nuclease activity of spCas9 and new Cas9 proteins from the IIB class, demonstrating how different Cas9 variants produce distinct indel profiles at their target sites.</p> <p>In cancer research, cells often lose the function of one DNA repair pathway, forcing them to rely on the remaining pathways to support their rapid replication. This dependency makes overactive DNA repair pathways promising targets for cancer drug discovery. Using RIMA, we can measure the overall DNA repair capacity of cells, validate the roles of key proteins in DNA repair pathways, and identify compounds that inhibit overactive repair mechanisms.</p> <p>For further details, see my publication in <em>Nucleic Acids Research</em> <a href="https://academic.oup.com/nar/article/46/16/8417/5055824" rel="external nofollow noopener" target="_blank">here</a>.</p> </article> <h2>References</h2> <div class="publications"> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/rima-480.webp 480w,/assets/img/publication_preview/rima-800.webp 800w,/assets/img/publication_preview/rima-1400.webp 1400w," type="image/webp" sizes="200px"> <img src="/assets/img/publication_preview/rima.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="rima.png" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> </div> <div id="taheri2018decoding" class="col-sm-8"> <div class="title">Decoding non-random mutational signatures at Cas9 targeted sites</div> <div class="author"> Amir Taheri-Ghahfarokhi, Benjamin J M Taylor, Roberto Nitsch, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Anders Lundin, Anna-Lina Cavallo, Katja Madeyski-Bengtson, Fredrik Karlsson, Maryam Clausen, Ryan Hicks, Lorenz M Mayr, others' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Nucleic acids research</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li></ol> </div> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Amir Taheri-Ghahfarokhi. </div> <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WRM6X7TS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?70d799092f862ad98c7876aa47712e20"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>